dev-libs/libqtxdg/Manifest

```diff
@@ -1 +1,2 @@
 DIST libqtxdg-2.0.0.tar.xz 62404 BLAKE2B dc84654693c6504c36633394ce4bafb46705b45ac3e7576b03b92710603c86375fd70d5ea76a2864e498c5b33728b02d215da5b65eb3728218ff97defd883bf0 SHA512 f4e32346eddedd31195093214a45e9831742ee886a14b5d7ba1d640a91e05473b9b8adc7bca9970090f1fa8187597cd10796c32ecea4ac02229e9869cd4f1db3
```

Comment:
wrong commit message

Reply:
Oh, this should be fixed now. Unless I changed it to the wrong thing. XD